src/community/features-templating/features-templating-core/src/main/java/org/geoserver/featurestemplating/builders/jsonld/JSONLDIteratingBuilder.java

```diff
@@ -1,41 +0,0 @@
-package org.geoserver.featurestemplating.builders.jsonld;
-
-import java.util.List;
-import org.geoserver.featurestemplating.builders.impl.IteratingBuilder;
-import org.geoserver.featurestemplating.builders.impl.TemplateBuilderContext;
-import org.xml.sax.helpers.NamespaceSupport;
-
-public class JSONLDIteratingBuilder extends IteratingBuilder {
-
-    public JSONLDIteratingBuilder(String key, NamespaceSupport namespaces) {
-        super(key, namespaces);
-    }
-
-    @Override
-    protected boolean canWrite(TemplateBuilderContext context) {
```

Comment:
I guess this canWrite method should go in the IteratingBuilder as done for the others

Reply:
Good catch

Reply:
Actually wait, I moved it... but it turned out, it was exactly equal to the one already in the iterating builder already. That left me baffled, but turns out it's code you set there in 2020:

![image](https://user-images.githubusercontent.com/325433/110965826-e0fb7700-8354-11eb-950a-0859b35e8990.png)

Reply:
my apologizes @aaime I did not see it, then pr is fine for me